core/src/main/scala/org/apache/spark/h2o/backends/SharedBackendConf.scala

```diff
@@ -380,4 +399,7 @@ object SharedBackendConf {
 
   /** Extra properties passed to H2O client during startup. */
   val PROP_CLIENT_EXTRA_PROPERTIES = ("spark.ext.h2o.client.extra", None)
+
+  /** Path to Java KeyStore file used for the internal SSL communication. */
+  val PROP_HADOOP_CONF = ("spark.ext.h2o.hadoof_conf", None)
```

Comment:
nit:  I'm not sure whether we should use camel case here. `spark.ext.h2o.hadoopConf` It's rather a question to think about. The conventions seems to be inconsistent.

Reply:
There is `spark.ext.h2o.internal_security_conf` or `spark.ext.h2o.internal_secure_connections"` and many others. I'm happy to follow any convention. No problem here.

Reply:
Hm, I would use the variant with `_` for now even though it is not the best, but at least consistent